apps/storefront/src/pages/ShoppingListDetails/index.test.tsx

```diff
@@ -112,6 +113,79 @@ const b2bCompanyWithShoppingListPermissions = buildCompanyStateWith({
   ],
 });
 
+const buildSearchProductsResponseWith = builder<SearchB2BProductsResponse>(() => ({
+  data: {
+    productsSearch: [
+      {
```

Comment:
🤔 isn't this something we should build using a helper builder and bulk? or is it always returning 1 result?

Reply:
yeah, it was hardcoded as a single result and because no tests were using it, it did't really matter

I've now split it out to have its own builder, to use bulk, and to be proven useful by a few new tests which require it:

- https://github.com/bigcommerce/b2b-buyer-portal/pull/328/commits/8c4399dae3314de017495b6864d024042eb294be